### PR TITLE
HRIN-414: Public meeting

### PR DESCRIPTION
**Pull request by martinyde**
https://jira.itkdev.dk/browse/HRIN-414

### CODE REVIEW

File: config/sync/core.extension.yml [hunk 1]
```diff
@@ -44,6 +47,7 @@ module:
   itk_azure_video: 0
   itk_cookie_message: 0
   itk_media_entity: 0
+  itk_pretix: 0
```
Comment (rimi-itk) [inline config/sync/core.extension.yml]: Are we missing `itk-dev/itk_pretix` in `composer.json`?

File: config/sync/itk_pretix.pretixconfig.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+itk_pretix: null
+_core:
+  default_config_hash: peWh3JrwZ60PoKUGdO3_bHBdvmb6HQb6LscjOgivUjc
+langcode: da
+api_token: '1234567'
```
Comment (rimi-itk) [inline config/sync/itk_pretix.pretixconfig.yml]: We should not commit our api token. We should probably use tricks similar to the ones used in https://github.com/itk-dev/hoeringsportal/blob/master/config/sync/system_status.settings.yml.

File: web/modules/custom/hoeringsportal_content_access/hoeringsportal_content_access.module [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ */
+function hoeringsportal_content_access_node_access(NodeInterface $node, $operation, AccountInterface $account) {
+  $bundle = $node->bundle();
+  if ($bundle == 'project' && $operation == 'view') {
+    $access = new AccessResultForbidden();
+  }
+  else {
+    $access = new AccessResultNeutral();
+  }
+  return $access;
```
Comment (rimi-itk) [inline web/modules/custom/hoeringsportal_content_access/hoeringsportal_content_access.module]: This should be `return ($bundle == 'project' && $operation == 'view') ? new AccessResultForbidden() : new AccessResultNeutral();`.

File: config/sync/core.extension.yml (hunk 1 above)
Comment (martinyde) [reply config/sync/core.extension.yml]: Added to composer.json and composer.lock

File: config/sync/itk_pretix.pretixconfig.yml (hunk 2 above)
Comment (martinyde) [reply config/sync/itk_pretix.pretixconfig.yml]: Changed config

File: web/modules/custom/hoeringsportal_content_access/hoeringsportal_content_access.module (hunk 3 above)
Comment (martinyde) [reply web/modules/custom/hoeringsportal_content_access/hoeringsportal_content_access.module]: Changed return

**Pull request merged**